test/unit/core/detail/int_types_test.cpp

```diff
@@ -6,7 +6,8 @@
 // -----------------------------------------------------------------------------------------------------
 
 #include <gtest/gtest.h>
-#include <seqan3/core/detail/int_types.hpp>
+
+#include <seqan3/utility/detail/int_types.hpp>
```

Comment:
In general, now that I looked at the file, `int_types.hpp` isn't very expressive. Can we give the file a better name, now that we rename it anyway? As I see it, there are no int_types defined but rather some traits of integers? What about `int_traits.hpp`? I'll post something on Gitter.